boards/adafruit-feather-nrf52840-express.yaml

```
@@ -0,0 +1,44 @@
+boards:
+  # different names in use: "Adafruit Feather nRF52840 Express" on shop as of
+  # 2025-12-14, whereas schematics dating from 2025-05-05 label it as "Adafruit
+  # nRF52840 Bluefruit Feather Express". The board itself does not have any
+  # "express" text on it at all.
+  adafruit-feather-nrf52840-express:
+    chip: nrf52840
+    flags:
+      - has_usb_device_port
+    leds:
+      # FIXME: led0/led1 names are purely synethetic to work with the current
```

```suggestion
      # FIXME: led0/led1 names are purely synthetic to work with the current
```
Added as a suggestion to `typos` upstream.